apps/edxapp/templates/_volumes/exports.yml.j2

```
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: edxapp-pvc-export
+  namespace: "{{ project_name }}"
+  labels:
+    app: edxapp
+    version: "{{ edxapp_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+spec:
+  accessModes:
+  - ReadWriteMany
```

For more formatting consistency (we also have to format other files in a another PR...):

```suggestion
    - ReadWriteMany
```